.github/workflows/windows.yml

```diff
@@ -5,12 +5,19 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
         ruby: [ '2.4', '2.5', '2.6', '2.7' ]
```

Comment:
Don't you update it?

Reply:
Fixed in https://github.com/fluent/fluent-plugin-rewrite-tag-filter/pull/111/commits/eeca8ba0e7ee03c1e92a3ce1e8278717856c0a66.